tron/mesos.py

```diff
@@ -366,6 +378,7 @@ def create_task(
             volumes=volumes,
             uris=uris,
             offer_timeout=self.offer_timeout,
+            task_id_override=task_id,
```

Comment:
Yeah, I was trying to avoid assuming anything about how the task_id is constructed by TaskConfig, in Tron. Maybe I could add a method to TaskConfig like `set_task_id(self, task_id)` so the logic would be contained there?

Reply:
Updated to pass in name and uuid.